Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Base image for ROS Noetic on Ubuntu 20.04 Focal
+FROM ros:noetic-ros-base-focal
+
+COPY . /opt/barracuda-thruster-output-server
+
+# Set working directory
+WORKDIR /opt
+
+# Install dependencies for the IMU sensor and update other tools
+RUN apt-get update && apt-get install -y \ 
+    udev \
```

Comment:
dependencies copied from a previous dockerfile should get removed if they aren't used in this container